opni-insights-service/app/main.py

```diff
@@ -3,6 +3,7 @@
 
 # Third Party
 from elasticsearch import Elasticsearch
```

Comment:
Shouldn't we use AsyncElasticsearch?

Reply:
I think using just Elasticsearch is okay for this.

Reply:
for example: `es_instance.count()` is called in `get_overall_breakdown()` which is used by `async def index_overall_breakdown(start_ts: int, end_ts: int):`.
So we should either use AsyncES, or define `index_overall_breakdown()` as non-async function?